### DISCUSSION TITLE
Using ANTs for registering images with artifact.

### DISCUSSION THREAD

**Discussion by willd4000** [ANTsX/ANTs#1474] [Q&A]
Hi, I am currently trying to register an MRI image with a minor artifact to a reference image without the artifact via ANTs. The reference image, however, has a background of different intensity compared with the artifact image. 

I have recently tried the antsRegistrationSyNQuick script with the rigid transformation setting and the histogram matching setting turned on. 4s14 is the artifact image and 12s14 is the reference image. The result did not match the background intensity of the artifact image sufficiently for my project. I was aiming for the white background intensity around the artifact (left cell of the image) to be less than 30% different when compared to the reference background. 

I am wondering if anyone has any recommendations for scripts or other settings to try. Thank you. 

![Reference_Image](https://user-images.githubusercontent.com/121213821/210108555-07dc16e4-2235-4108-809c-76835db299ed.png)
![Artifact_Image](https://user-images.githubusercontent.com/121213821/210108557-2b54ffdf-a4e1-4270-9cb3-cf60adb6d0ec.png)
[myRegOutput4s14.txt](https://github.com/ANTsX/ANTs/files/10326157/myRegOutput4s14.txt)

**Comment by gdevenyi** [comment]
Generally with abnormalities, your options are either to mask out the region you want to ignore, or to try to do a lesion filling of the artifact region so the images are more similar in their content.
